templates/default/ckan-monitor-job-queue.sh.erb

```diff
@@ -13,3 +14,33 @@ else
     MAX_AGE=0
 fi
 aws cloudwatch put-metric-data --region <%= node['datashades']['region'] %> --namespace CKAN --metric-data "MetricName=job_queue_size,Dimensions=[{Name=Application,Value=<%= node['datashades']['app_id'] %>},{Name=Environment,Value=<%= node['datashades']['version'] %>}],Value=$JOB_COUNT" "MetricName=max_job_age,Dimensions=[{Name=Application,Value=<%= node['datashades']['app_id'] %>},{Name=Environment,Value=<%= node['datashades']['version'] %>}],Value=$MAX_AGE"
+
+# Default queue
```

Comment:
This should probably be extracted to a function. Also, which queue will the weekly archiver run use? Perhaps we should monitor the 'priority' queue.

Reply:
i left out one of the queues as you can deduce it from the others

Reply:
weekly archiver uses the bulk queue, priority is only used by resource updates/package updates

Reply:
Fair enough. I did the refactor, though, so adding priority monitoring was only a one-liner.